### PR TITLE
Add notebook doc img2img

### PR DESCRIPTION
**Pull request by yiyixuxu**
This PR is the first example of how we can integrate the notebook into the doc page itself

for https://huggingface.co/docs/diffusers/using-diffusers/img2img

there are a lot of other doc pages that have a link to a colab notebook - plan to integrate them all after I get feedback for this one (so that we don't have to maintain these notebooks separately) 

for example 
https://huggingface.co/docs/diffusers/stable_diffusion
https://huggingface.co/docs/diffusers/using-diffusers/inpaint
https://huggingface.co/docs/diffusers/using-diffusers/rl
https://huggingface.co/docs/diffusers/using-diffusers/other-modalities

### CODE REVIEW

File: docs/source/en/using-diffusers/img2img.mdx [hunk 1]
```diff
@@ -12,7 +12,17 @@ specific language governing permissions and limitations under the License.
...
 
-The [`StableDiffusionImg2ImgPipeline`] lets you pass a text prompt and an initial image to condition the generation of new images.
+[[open-in-colab]]
+
+The [`StableDiffusionImg2ImgPipeline`] lets you pass a text prompt and an initial image to condition the generation of new images. This tutorial shows how to use it for text-guided image-to-image generation with Stable Diffusion model.
+
+Before you begin, make sure you have all the necessary libraries installed:
+
+```bash
+!pip install diffusers==0.11.1 transformers ftfy accelerate
```
Comment (patrickvonplaten) [inline docs/source/en/using-diffusers/img2img.mdx:22]: We should always use the most up-to-date version of `diffusers` in our docs
```suggestion
!pip install diffusers transformers ftfy accelerate
```

File: docs/source/en/using-diffusers/img2img.mdx [hunk 2]
```diff
@@ -12,7 +12,17 @@ specific language governing permissions and limitations under the License.
 
 # Text-Guided Image-to-Image Generation
 
-The [`StableDiffusionImg2ImgPipeline`] lets you pass a text prompt and an initial image to condition the generation of new images.
+[[open-in-colab]]
```
Comment (patrickvonplaten) [inline docs/source/en/using-diffusers/img2img.mdx:15]: Where is the corresponding generated google colab?

**Review by patrickvonplaten — COMMENTED** [review]
Can you also link the google colab that is automatically generated from this doc?

**Comment by yiyixuxu** [comment]
@patrickvonplaten I think with this line `[[open-in-colab]]` - the doc-builder will automatically convert it to a notebook, push to `huggingface/notebooks` and link it once the PR is merged 

Maria and Steven told me so -  we can merge this one and see what happens.

File: docs/source/en/using-diffusers/img2img.mdx [hunk 3]
```diff
@@ -21,25 +31,83 @@ from PIL import Image
...
+image = pipe(prompt=prompt, image=init_image, strength=0.5, guidance_scale=7.5, generator=generator).images[0]
+image
+```
+
+![img](https://huggingface.co/datasets/YiYiXu/test-doc-assets/resolve/main/image_2_image_using_diffusers_cell_14_output_1.jpeg)
+
+
+As you can see, when using a lower value for `strength`, the generated image is more closer to the original `image`
+
+Now let's use a different scheduler - [LMSDiscreteScheduler](https://huggingface.co/docs/diffusers/api/schedulers#diffusers.LMSDiscreteScheduler)
```
Comment (patrickvonplaten) [inline docs/source/en/using-diffusers/img2img.mdx:94]:
```suggestion
Now let's use a different scheduler - [LMSDiscreteScheduler](https://huggingface.co/docs/diffusers/api/schedulers#diffusers.LMSDiscreteScheduler)
```

Let's use maybe a different scheduler here, such as DPMSolverMultistepScheduler -> LMS requires another dependency and not the best scheduler anymore

**Pull request merged**